### PR TITLE
Modal for switching bed fixed

### LINKED ISSUE #2458: Switch Bed Working Issue
hoiih

- the UI is top of the page, so can't see the patient details
- the Switch bed is not auto fetching the current time and date to the date of shifting

### PR DESCRIPTION
**Pull request by ishanExtreme**
### Issue Fixed
fixes #2458 

### Updates

- Added small loader as original loader was making modal unresponsive
- Shifted modal position
- Made modal responsive

### Demo

https://user-images.githubusercontent.com/39593226/169654547-dccda974-af7b-4e7f-a6a0-43f3b4b22566.mp4

### CODE REVIEW

**Comment by nihal467** [comment]
eoifjw

@ishanExtreme @gigincg test failed, 
- the UI is very small
- the date is not auto fetching
- the date is not fully visible

**Comment by ishanExtreme** [comment]
@nihal467 
Can you give more details about date auto fetching issue.

**Comment by nihal467** [comment]
@ishanExtreme @gigincg when we click the switch bed, button, the date of shifting column should be auto-filled with the current time and date

**Comment by ishanExtreme** [comment]
@nihal467 
Fixed above issues

**Comment by nihal467** [comment]
epjof

@ishanExtreme @gigincg test failed, when we click the switch bed button, the current date and time are not been auto fetched

**Comment by ishanExtreme** [comment]
@nihal467 
After the new changes, I pushed 4 days a go, date of shift is now fetching current date
Screenshot from 2022-05-28 16-51-39

**Comment by ishanExtreme** [comment]
@nihal467 
Nelify preview build is not showing the latest changes

**Comment by rithviknishad** [comment]
Netlify preview does show up with current date time for me.

image

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/Consultations/Beds.tsx [hunk 1]
```diff
@@ -10,21 +10,35 @@ import Loading from "../../Common/Loading";
...
 import { BedSelect } from "../../Common/BedSelect";
 import { Button, InputLabel } from "@material-ui/core";
+import CircularProgress from "@material-ui/core/CircularProgress";
 import { TextInputField } from "../../Common/HelperInputFields";
 import CheckCircleOutlineIcon from "@material-ui/icons/CheckCircleOutline";
 import moment from "moment";
 
+const formatDateTime: () => string = () => {
+  const current = new Date();
+  console.log(current);
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/Beds.tsx]: do remove console logs

File: src/Components/Facility/Consultations/Beds.tsx [hunk 2]
```diff
@@ -82,13 +96,23 @@ const Beds = (props: BedsProps) => {
     }
   };
 
-  if (isLoading) return <Loading />;
+  if (isLoading) {
+    // small loader for modal
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/Beds.tsx]: do remove the comment

File: src/Components/TeleIcu/Patient/InfoCard.tsx [hunk 3]
```diff
@@ -15,18 +15,23 @@ export default function TeleICUPatientInfoCard({
   patient,
 }: ITeleICUPatientInfoCardProps) {
   const [open, setOpen] = useState(false);
-  console.log(patient);
+  // console.log(patient);
```
Comment (khavinshankar) [inline src/Components/TeleIcu/Patient/InfoCard.tsx]: do remove this comment

**Review by khavinshankar — CHANGES_REQUESTED** [review]
please do remove the comments and console logs

**Review by khavinshankar — APPROVED** [review]

File: src/Components/Facility/Consultations/Beds.tsx (hunk 1 above)
Comment (khavinshankar) [inline src/Components/Facility/Consultations/Beds.tsx]:
```suggestion
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**